### DISCUSSION TITLE
Assigning ID on material

### DISCUSSION THREAD

**Discussion by Jina-Kim-1108** [l3atbc-datadog/pushkin#112] [General] [answered]
Hi! Thanks so much for your reply in advance!

I have a question for assigning ID on each trial. 
Below is the code of my study. Since I randomize the order of ‘text’ here, I would like to assign ID from I to 4 on each text so that I can sort all participants data accordingly.

For cloze plug-in, the data is answers. Can I also get ‘check_answers’ value for each trial?
Here is the code

var survey = {
                    type: 'cloze',
                    text: jsPsych.timelineVariable('text'),           
                    check_answers: true,
                    button_text: 'Next'
                    };

                var actual_survey ={
                    timeline: [survey],
                    timeline_variables: [
                {text: "In the rain forest," + "<p>lion is only % a %."},
                {text: "In the river," + "<p>I saw fish eats % b %."},
                {text: "Practice 3," + "<p>Praceice% b %."},
                {text: "Practice 4," + "<p>Praceice% b %."},
                {text: "Practice 5," + "<p>Praceice% b %."}
                ],
                randomize_order : true
                }

**Comment by restmegan** [comment, marked as answer]
I think this question would be better suited to the [jsPsych forum](https://github.com/jspsych/jsPsych/discussions)

**Reply by Jina-Kim-1108** [reply]
Thanks for letting me know :)

**Comment by restmegan** [comment]
Hi @Jina-Kim-1108 ! I just wanted to check in to see how Pushkin has been going for you. The Pushkin team is online today answering questions so if there's anything you need help with please let us know!
